flexmeasures/ui/templates/crud/asset.html

```diff
@@ -64,7 +69,7 @@
                 <button id="asset-status-button" class="btn btn-sm btn-responsive" type="submit">Asset status</button>
             </form>
             <div class="sidepanel-container">
-                <div class="left-sidepanel-label">Select dates</div>
+                <div class="left-sidepanel-label" style="margin-left: 20px;">Select dates</div>
```

Comment:
This style content can also go to flexmeasures.css

Reply:
sorted